tests/torchtune/models/test_lora_llama2.py

```diff
@@ -109,17 +110,22 @@ def get_lora_llama2(
         # To make final outputs less trivial
         if reset_norm:
             model.norm = nn.Identity()
-        fixed_init_model(model)
+
+        # Some tests don't rely on fixed_init, NF4Tensor does not have op
+        # coverage to cover ops done in fixed_init (i.e. arange). So only
+        # run fixed_init if needed.
```

Comment:
Couple questions: (a) is the op needed to support arange a lot of work to add an override for? (b) can we deterministically initialize tensors in another way without using an unsupported op? E.g. just use regular Python range and populate in tensor. I know the QLoRA tests don't need it at this moment but it's kind of a weird rough edge for us to leave laying around.

Reply:
So I'm chatting with the ao team about what to do in the longer term here. Adding support for `arange` won't be similar to the work we did for torch_dispatch-based op registrations, since `arange` is a factory function (produces torch tensors) rather than an op on existing tensors. 

> t's kind of a weird rough edge for us to leave laying around

Totally agree with this - this is just to unblock and I'll have some follow up thoughts asap after chatting with ao folks.

Reply:
>  (b) can we deterministically initialize tensors in another way without using an unsupported op? E.g. just use regular Python range and populate in tensor

Sure, let me try to do this

Reply:
Ok, worked around this by forcing the fixed init when initializing QLoRA models to be bf16.